packages/frontend/src/components/wallet-migration/metrics.js

```diff
@@ -24,7 +24,7 @@ export const initAnalytics = () => {
             return;
         }
       
-        const rudderAnalyticsKey = IS_MAINNET
```

Comment:
should we fix `IS_MAINNET` rather than inlining the correct logic here?

Reply:
I'm not sure. `IS_MAINNET` is being used in many other places with existing logic. If we make change here just to make it work for `RudderStack` it could break on other part. 

Also I think it was just wrong on using `IS_MAINNET` here. (`IS_MAINNET` is also considered as `true` for `staging`) Which is why I decided to change the condition